src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksArrSlice from './books/booksArrSlice';
+
+const store = configureStore({
+  reducer: {
+    bookstore: booksArrSlice,
+  },
+});
+
+export default store;
```

Comment:
- [ ] You are yet to import the categories reducer as you have imported the books reducer. Kindly do the same for categories.